src/storekit.cs

```diff
@@ -617,11 +647,13 @@ interface SKCloudServiceController {
 		NSString StorefrontCountryCodeDidChangeNotification { get; }
 	}
 
+#if !WATCH
 	[iOS (10,1)]
 	[NoTV] // __TVOS_PROHIBITED
 	[BaseType (typeof(UIViewController))]
 	interface SKCloudServiceSetupViewController
 	{
+#endif // !WATCH
```

Comment:
^ I think your `#endif` should be after the end of the type
(or I'm missing something)